uint32 msgId 
double vehicleSpeed
double engineSpeed 
bool driverdoor_open 
bool FrontPassengerDoorOpen 
bool RearLeftDoorOpen 
bool RearRightDoorOpen 
uint32 FrameTickCountSysReadTime
